### PR TITLE
fix: broken vxs and readme

### PR DESCRIPTION
**Pull request by ehxxn**

### CODE REVIEW

File: examples/vxs/app/_layout.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { TamaguiProvider, isWeb } from 'tamagui'
 import { Stack } from 'vxs'
 import config from '../src/tamagui.config'
 // @ts-ignore idk why only cli watch is getting mad at this
-import { TestComponent } from '~/components/TestComponent'
+import { TestComponent } from '../components/TestComponent'
```
Comment (natew) [inline examples/vxs/app/_layout.tsx:8]: this should work tho, works in the tamagui site in the repo, would rather fix than revert

Comment (ehxxn) [reply examples/vxs/app/_layout.tsx:8]: ok so the issue was this line 
```
resolve: {
    '~': import.meta.dirname
}
```
which is undefined, only `import.meta.url` is there. I think support for `dirname` added to Node `20.11`
need to check that

Comment (ehxxn) [reply examples/vxs/app/_layout.tsx]: ok used `import.meta.url` instead to work with older versions of node

Comment (natew) [reply examples/vxs/app/_layout.tsx]: so actually the runtime was working for me it was the types which were complaining weirdly enough. i use node 22, we should just add that to the .node-version i think

**Pull request closed without merging**